### PR TITLE
Adding Console logic to enable Console Dynamic Velocity Profiles capa…

### PR DESCRIPTION
**Pull request by arunabh-c**
…bility

New option added to console script to enable process noise motion modes toggle

### CODE REVIEW

**Comment by arunabh-c** [comment]
Corresponding Starling PR: https://github.com/swift-nav/starling/pull/1975
Corresponding PFWP PR: https://github.com/swift-nav/piksi_firmware_private/pull/2719

File: piksi_tools/console/settings.yaml [hunk 1]
```diff
@@ -404,6 +404,21 @@
...
     attempted by the receiver.
 
+- group: solution
+  name: process_noise_motion_mode
+  expert: false
+  type: enum
+  units: N/A
+  default value: 'Default'
+  readonly: false
+  enumerated possible values: Default, Automotive, Static
```
Comment (pgrgich) [inline piksi_tools/console/settings.yaml]: As for the other PR, I'm a bit unsure what our options should be. If we want to keep the current configuration with high process noise, we should probably just name it something that makes it obvious what it is (or perhaps "UAV", if that's what the UAV use case will use). It sounds like "Static" has changed a bit, but why does the static case have non-zero process noise? Is it assumed that tectonic motion needs to be considered? If it's more than that, I'd think the name should be changed to "Low dynamics" or something.

**Comment by anth-cole** [comment]
Can we make sure this is coordinates with Alex? This is the customer facing piece so we should get product input.

I'm not a fan of the setting name, it requires the user to know what the process noise is, what about 'dynamic motion model' or something?

File: piksi_tools/console/settings.yaml [hunk 2]
```diff
@@ -404,6 +404,21 @@
     for each timestamp. When "No DGNSS" is chosen, no differential output will be
     attempted by the receiver.
 
+- group: solution
+  name: process_noise_motion_mode
```
Comment (scarcanague) [inline piksi_tools/console/settings.yaml]: Have you discussed with Alex on this? `dynamic motion model'` sounds indeed more customer friendly

File: piksi_tools/console/settings.yaml [hunk 3]
```diff
@@ -404,6 +404,21 @@
...
+  expert: false
```
Comment (scarcanague) [inline piksi_tools/console/settings.yaml]: That should be true, I dont think we want the setting in the default intially

File: piksi_tools/console/settings.yaml [hunk 4]
```diff
@@ -404,6 +404,21 @@
...
+  type: enum
+  units: N/A
+  default value: 'Default'
+  readonly: false
+  enumerated possible values: Default, Automotive, Static
+  Description: Selects the process noise variance of position, velocity & acceleration in the Horizontal &
```
Comment (scarcanague) [inline piksi_tools/console/settings.yaml]: We probably also want something a little more customer friendly for the description

**Comment by arunabh-c** [comment]
> As with the other PRs, I'm not sure what the status is for naming. Can you confer with Sebastien, Alex and anyone else they consider needs to be involved in order to get to a conclusion?

waiting to hear back from alex on final version for names and description.

**Comment by pgrgich** [comment]
For the purposes of our first cut, shall we go with something like:
Model name: `dynamic motion model`
Model options: `HIGH_DYNAMICS`, `HIGH_HORIZONTAL_DYNAMICS` and `LOW_DYNAMICS` (replacing `DEFAULT`, AUTOMOTIVE` and `STATIC`).
Model descriptions: `High dynamics - high process noise in all axes`, `High horizontal dynamics - high process noise in horizontal plane and low process noise in vertical axis` and `Low dynamics - low process noise in all axes`
Happy for this to be changed, but I think we need something to iterate on.

**Pull request merged**